### PR TITLE
1196 add dose to medium and let to medium

### PR DESCRIPTION
**Pull request by Derstilon**
requires https://github.com/yaptide/converter/pull/149 to be merged

### CODE REVIEW

**Comment by grzanka** [comment]
Duplication of Quantities stopped working on this branch:

https://github.com/yaptide/ui/assets/7374733/7f4e4738-e8d9-40be-85d9-ee6a06c91025

It is working perfectly well on the main branch on github pages deploy:



https://github.com/yaptide/ui/assets/7374733/0d5277a2-fef1-484b-9173-f09d1ba30d36

File: src/config/ConfigService.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { createGenericContext } from '../services/GenericContext';
 import { UppercaseObjectKeys, ValidateKeysTuple } from '../types/TypeTransformUtil';
 import { snakeToCamelCase } from '../util/Notation/Notation';
 
-const BACKEND_URL = process.env.REACT_APP_BACKEND_URL ?? 'http://localhost:5000';
+const BACKEND_URL = process.env.REACT_APP_BACKEND_URL ?? 'https://localhost:8443';
```
Comment (grzanka) [inline src/config/ConfigService.tsx:7]: Why this has been changed here ? According to the docs the UI should talk to http backend, see https://yaptide.github.io/deploy/

File: src/services/ShSimulatorService.tsx [hunk 2]
```diff
@@ -417,7 +417,7 @@ const ShSimulation = ({ children }: GenericContextProviderProps) => {
 			authKy
 				.delete(getEndpointFromSimulationInfo(info), {
 					signal,
-					searchParams: camelToSnakeCase(info.jobId)
+					searchParams: camelToSnakeCase({ jobId: info.jobId })
```
Comment (grzanka) [inline src/services/ShSimulatorService.tsx:420]: I think this PR should be rebased, as this change has already been merged:
https://github.com/yaptide/ui/pull/1232/files

File: src/ThreeEditor/examples/ex1.json [hunk 3]
```diff
@@ -261,13 +261,15 @@
...
             "name": "BLACK HOLE",
+            "sanitizedName": "black_hole",
             "icru": 0,
             "density": 1,
             "color": 196613
           },
           {
             "uuid": "49ce56b9-ee48-4640-88d4-bcfd660d8c0e",
             "name": "WATER, LIQUID",
+            "sanitizedName": "water_liquid",
```
Comment (grzanka) [inline src/ThreeEditor/examples/ex1.json:272]: Does this change imply that you need to bump JSON standard version ?

File: src/config/ConfigService.tsx (hunk 1 above)
Comment (Derstilon) [reply src/config/ConfigService.tsx:7]:
```suggestion
const BACKEND_URL = process.env.REACT_APP_BACKEND_URL ?? 'http://localhost:5000';
```

**Comment by grzanka** [comment]
Is it still draft or ready for review?

**Comment by grzanka** [comment]
Can you update examples and bump examples version to 0.11 ? Otherwise I see

image

**Comment by grzanka** [comment]
Converter changes are merged now, maybe you could update the converter in this PR as well and fix the tests ?

**Comment by Derstilon** [comment]
> Is it still draft or ready for review?

I've merged new converter and change status to ready

**Comment by grzanka** [comment]
> > Is it still draft or ready for review?
> 
> I've merged new converter and change status to ready

OK, I will check it.

**Comment by grzanka** [comment]
@Derstilon take a look as well at https://github.com/yaptide/yaptide/pull/639
I think that example JSON files in backend test suite needs update as well

**Review by grzanka — DISMISSED** [review]
OK

**Review by grzanka — CHANGES_REQUESTED** [review]
Needs https://github.com/yaptide/yaptide/pull/639 to be merged first

**Pull request merged**